tests/integration/backup_tests/test_backups.py

```diff
@@ -94,7 +106,7 @@ async def test_build_and_deploy(ops_test: OpsTest) -> None:
     if not application_name:
         application_name = await ha_helpers.deploy_and_scale_application(ops_test)
 
-    db_app_name = await ha_helpers.get_application_name(ops_test, DATABASE_APP_NAME)
+    db_app_name = await get_app_name(ops_test) or DATABASE_APP_NAME
```

Comment:
IIRC `get_app_name` will retrieve the newly deployed charm correctly, so there is no need for ` or DATABASE_APP_NAME` 
```suggestion
    db_app_name = await get_app_name(ops_test) 
```

Reply:
done